.npmignore

```diff
@@ -0,0 +1,2 @@
+user-config.yml
+node_modules/
```

Comment:
I don't like having this file as it's horribly easy to forget to add `user-config.yml` which could cause us to publish secrets... at least in some modules... Suggestions for a good pattern to ensure that we do this right are welcome...